common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -32,6 +33,16 @@ def setUp(self):
             self.browser, username=self.USERNAME, password=self.PASSWORD, email=self.EMAIL
         ).visit().get_user_id()
 
+    def assert_event_emitted_num_times(self, setting, num_times):
+        """
+        Verify a particular user settings change event was emitted a certain
+        number of times.
+        """
+        # pylint disable=no-member
+        super(AccountSettingsTestMixin, self).assert_event_emitted_num_times(
```

Comment:
Must: please also check that the referer ends with /account/settings. We are relying on the fact that the referer will provide information about which page initiated the event.